tests/system/libraries/systemTestSpy.py

```diff
@@ -0,0 +1,207 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2018 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+"""This file provides spy and robot library behaviour for NVDA system tests.
+It is copied into the (system test specific) NVDA profile directory. It becomes the '__init__.py' file as part of a
+package. This allows us to share utility methods between the global plugin and the nvdaRobotLib library.
+"""
+import globalPluginHandler
+import threading
+from systemTestUtils import _blockUntilConditionMet
+from logHandler import log
+from time import clock as _timer

```

Comment:
why does this name get changed to _timer instead of _clock

Reply:
No real reason, except that originally I was importing `from timeit import default_timer as _timer`